pkgs/os-specific/linux/aic8800/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  kernel,
+  kernelModuleMakeFlags,
+  dpkg,
+  aic8800-firmware,
+}:
+let
+  source = aic8800-firmware.passthru.source;
+in
+stdenv.mkDerivation (finalAttr: {
+  name = "aic8800";
+  version = source.version;
+  src = source.src;
+
+  hardeningDisable = [ "pic" ];
```

Comment:
Surprising that this is necessary given we don't use the compiler wrapper for kernel modules any more.